api/api/error_handler.py

```diff
@@ -55,6 +55,28 @@ def _cleanup_detail_field(detail: str) -> str:
     return ' '.join(str(detail).replace("\n\n", ". ").replace("\n", "").split())
 
 
+async def expect_failed_error_handler(request: ConnexionRequest, exc: ExpectFailedException) -> ConnexionResponse:
+    """Handler for the 'Expect' HTTP header.
+    
+    Parameters
+    ----------
+    request : ConnexionRequest
+        Incoming request.
+
+    Returns
+    -------
+    Response
+        HTTP Response returned to the client.
+    """
+    problem = {
+        "title": "Expectation failed",
+        "detail": "Unknown Expect",
+        "error": 417
+    }
```

Comment:
The detail field could be different depending on whether the user included a unknown expect or the Content-Length was exceeded:

```suggestion
    problem = {
        "title": "Expectation failed"
    }

    if exc.detail:
        problem['detail'] = exc.detail